assets/section-image-banner.css

```diff
@@ -158,6 +158,16 @@
   left: auto;
 }
 
+.banner__media-half.animate--fixed:first-child>img {
+  left: 0;
+  width: 50%;
+}
+
+.banner__media-half.animate--fixed:nth-child(2)>img {
+  left: 50%;
+  width: 50%;
+}
+
```

Comment:
Actually it's even more so specific to the admin when you view the mobile layout. Because you have a pretty tall browser window. When I test from my phone it's better because it's as tall. Could be an odd experience for merchants to understand 🤔

Reply:
@ludoboludo Can you record a video to explain what you observe? I am not sure I understand 🤔

Reply:
I think the [video](https://screenshot.click/17-35-n8sx8-gv6i8.mp4) shared in my first message shows what I'm observing. Basically the thing to know here is that a fixed element is placed based on the viewport itself not the element its in. 

So there can be some poor experiences in certain scenarios: 
- if the viewport height is quite high
- if you're viewing the site on big screen/wide browser window as the image has a max size of `3840px` wide
- depending on the image format, portrait/landscape, some example are more or less obvious
- if the image/section is at the bottom/top of the page, the visible part isn't going to show what you're actually trying to show in some occasions

[Here is a video recap of those examples](https://screenshot.click/20-18-6ufaz-nrvex.mp4) (it's pretty long 😬 7min)

Reply:
About **focal points**, I'm actually not sure what the solution would be here. In order for this effect to work for every section position, the height of the image must be `100vh`, otherwise you will have empty space at some point while scrolling.

Here's an example where I added `max-height: 50%` (arbitrary value) to limit the image heights:

https://user-images.githubusercontent.com/37168033/233444863-cb493532-8253-4afa-8be4-cc93989453a2.mp4

It looks perfect for the first section, but as you scroll, you realize that the images MUST be 100% height for the effect to work properly. I would say the is almost an expected side-effect of turning this setting on: it will cause the focal point to not show up by default in some situations. Should we have any help text to let merchants know of that? If you have any alternative suggestions to solve this issue, let me know.

As for **image sizes**, I think you just made a great point in the video that we may need to consider bumping the resolution we serve them. I'm pretty sure that right now it's serving a lower quality image than we should.

Reply:
@ludoboludo I've updated the image sizes to 100% so there is no resolution drop. You can see a clear improvement in mobile and half-width images.

Here's the before & after:

<img width="360" alt="image" src="https://user-images.githubusercontent.com/37168033/234306905-1abb50c9-2abb-4be8-809e-21f07c6209de.png"><img width="360" alt="image" src="https://user-images.githubusercontent.com/37168033/234307061-b3b89206-dad1-456d-8c72-179943b86140.png">

<img width="1285" alt="image" src="https://user-images.githubusercontent.com/37168033/234307387-6644e8bd-6e1d-4193-9f75-1ca0b8fa7405.png">
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/37168033/234307523-25579f27-5027-429f-84da-7739433626a8.png">